Android/app/src/main/java/app/intra/sys/firebase/RemoteConfig.java

```diff
@@ -1,90 +0,0 @@
-/*
-Copyright 2019 Jigsaw Operations LLC
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-https://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-package app.intra.sys.firebase;
-
-import android.os.Build;
-import android.os.Build.VERSION;
-import android.os.Build.VERSION_CODES;
-import com.google.android.gms.tasks.Task;
-import com.google.android.gms.tasks.Tasks;
-import com.google.firebase.remoteconfig.FirebaseRemoteConfig;
-
-/**
- * Utility class for initializing Firebase Remote Config.  Remote Configuration allows us to conduct
- * A/B tests of experimental functionality, and to enable or disable features without having to
- * release a new version of the app.
- */
-public class RemoteConfig {
```

Comment:
Would it be useful to retain a generic version of this class for future experiments?

Reply:
OK, done.  I also added comments in the places where it would need to be initialized.